README.md

```diff
@@ -78,6 +78,7 @@ followed by a possibly unbounded number of `onNext` signals (as requested by `Su
 | <a name="term_responsivity">Responsivity</a> | Readiness/ability to respond. In this document used to indicate that the different components should not impair eachothers ability to respond. |
 | <a name="term_non-obstructing">Non-obstructing</a> | Quality describing a method which is as quick to execute as possible—on the calling thread. This means, for example, avoids heavy computations and other things that would stall the caller´s thread of execution. |
 | <a name="term_terminal_state">Terminal state</a> | For a Publisher: The point where `onComplete` or `onError` has been signalled. For a Subscriber: The point where an `onComplete` or `onError` has been received.|
+| <a name="term_nop">NOP</a> | No operation—does nothing.|

```

Comment:
Maybe:

"No OPeration - executing the action has no visible primary- or side-effects to the current or other threads."